pkg/connectors/utils/utils.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"net/http"
+
+	"github.com/rs/zerolog"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kclient "sigs.k8s.io/controller-runtime/pkg/client"
+	kconfig "sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	fybrikTLS "fybrik.io/fybrik/pkg/tls"
+)
+
+// GetHTTPClient returns an object of type *http.Client.
+func GetHTTPClient(log *zerolog.Logger) *http.Client {
+	scheme := runtime.NewScheme()
+	log.Info().Msg(fybrikTLS.TLSEnabledMsg)
+	err := corev1.AddToScheme(scheme)
```

Comment:
do we need to add schema here, we did it in the main files, didn't we ?

Reply:
I changed the code to pass down scheme from main.go.